pyproject.toml

```diff
@@ -71,49 +71,161 @@ line-length = 79
 indent-width = 4
 
 target-version = "py38"
+exclude = [
+  "CSV",
+  "ref.*?py",
+]
 
 [tool.ruff.lint]
-fixable = ["ALL"]
-preview = true
+fixable = []
+# preview = true
+# logger-objects = []
+task-tags = ["TODO", "FIXME", "XXX"]
 select = [
+  "A", # flake8-builtins
+  "ARG", # flake8-unused-arguments
+  "COM", # flake8-commas
```

Comment:
Does "A" not cover what we want to keep from flake? I dont use COM in my local rule set but maybe I should be?

Reply:
We don't break it, why not using COM?

I added a couple more to better cover flake8. Anything specific you would like to include?

Reply:
Was just curious if its redundant. I'm fine with including it though. Re: other includes - I have SIM and PERF in my own set but this will break too much. Otherwise this closely matches my config so I'm happy.

Reply:
@bnb32 , I just added those two. I had to ignore a few specific rules so it doesn't break with sup3r as it is. As I mentioned before, I suggest starting like that and later we can conform and remove the specific violations one by one, but it would work as it is now.

Reply:
I'm closing this one but feel free to re-open if want to continue this discussion.